src/fft.js

```diff
@@ -472,6 +472,17 @@ define(function (require) {
     }
     this.analyser.smoothingTimeConstant = s;
   };
+  
+  /**
+   * Changes the length of the resulting array, the bin size and in turn the fftSize of the analyser.
+   * 
+   * @method setBins
+   * @param {Number} bins    Must be a power of two between 16 and 1024.
+   */
+  p5.FFT.prototype.setBins = function(b) {
+    this.bins = b;
+    this.analyser.fftSize = b * 2;
```

Comment:
Maybe this was addressed in #176, but the idea is that if I call

`fft.setBins(100)`

then this works

`this.bins = 100`

but this throws an error 

`this.analyser.fftSize = 100 * 2;`

`Failed to set the 'fftSize' property on 'AnalyserNode': The value provided (100) is not a power of two.`